src/app/layout.tsx

```diff
@@ -37,13 +41,43 @@ export default function RootLayout({
 }: Readonly<{
   children: React.ReactNode;
 }>) {
+  const localTheme = useMemo(() => {
+    if (typeof window === 'undefined') return dark;
+    const theme = localStorage.getItem('current-theme');
+    if (theme === DARK_THEME) {
+      return dark;
+    } else {
+      return light;
+    }
```

Comment:
Como é feito o return no caso do _if_, o _else_ imediatamente se torna desnecessário.
Do mesmo modo, como é uma validação breve, e também não se encontra em um operador ternário (que resultaria no encadeamento de ternários, paia), o ideal seria implementar o _return_ diretamente.

```suggestion
    return theme === DARK_THEME ? dark : light;
```